src/main.ts

```diff
@@ -36,12 +37,24 @@ let rl = readline.createInterface({
 
 const waitForInput = () => {
   rl.question('Input ID:\n', async (idNum) => {    
-    const result = await axios.post('/checkin', {
-      mac_address: callMac(),
-      student_id: idNum,
-    });
     waitForInput()
+    sendCheckIn({
```

Comment:
Great idea to make this its own function. Also great job in moving `waitForInput` call above. Makes much more sense. Just know that we've now made software somewhat more reliable than the "checkin system" they have at the gym.